modules/cpmso-yaml-sample-cr.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-sample-cr_{context}"]
+= Sample YAML for a control plane machine set custom resource
+
+The base of the `ControlPlaneMachineSet` custom resource is structured the same way for all platforms.
+
+.Sample `ControlPlaneMachineSet` CR YAML file
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster <1>
+  namespace: openshift-machine-api
+spec:
+  replicas: 3 <2>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-machine-role: master
+      machine.openshift.io/cluster-api-machine-type: master
+  strategy:
+    type: RollingUpdate <3>
+  template:
+    machineType: machines_v1beta1_machine_openshift_io
+    machines_v1beta1_machine_openshift_io:
+      failureDomains:
+        platform: <platform> <4>
+        <platform_failure_domains> <5>
+      metadata:
+        labels:
+          machine.openshift.io/cluster-api-cluster: <cluster_id> <6>
+          machine.openshift.io/cluster-api-machine-role: master
+          machine.openshift.io/cluster-api-machine-type: master
+      spec:
+        providerSpec:
+          value:
+            <platform_provider_spec> <7>
+----
+<1> Specifies the name of the `ControlPlaneMachineSet` CR is `cluster`. Do not change this value.
+<2> Specifies the number of control plane machines. An {product-title} cluster has three control plane machines, so the `replicas` value is `3`. Horizontal scaling is not supported. Do not change this value.
+<3> Specifies the update strategy for the cluster. The allowed values are `OnDelete` and `RollingUpdate`. The default value is `RollingUpdate`. For more information about update strategies, see "Updating the control plane configuration".
```

Comment:
I didn't see anything about strategies, is this still to come?

Reply:
Yup, later section :)